kolibri/content/models.py

```diff
@@ -109,53 +109,60 @@ class Admin:
     def __str__(self):
         return self.title
 
-class MimeType(AbstractContent):
-    """
-    Normalize the "kind"(mimetype) of Format model
-    """
-    readable_name = models.CharField(max_length=50)
-    machine_name = models.CharField(max_length=100)
+class ContentKind(AbstractContent):
+    kind = models.CharField(max_length=200, blank=True, null=True)
 
-    class Admin:
-        pass
+    def __str__(self):
+        return self.kind
+
+class FileFormat(AbstractContent):
+    extension = models.CharField(primary_key=True, max_length=40)
+    # content_kind = models.ForeignKey(ContentKind, related_name='kind', blank=True, null=True)
 
     def __str__(self):
-        return self.readable_name
+        return self.extension
 
-class Format(AbstractContent):
-    """
-    The intermediate layer of the contentDB schema, defines a complete set of resources that is ready to be rendered on the front-end,
-    including the quality of the content.
-    Things it can represent are, for example, high_resolution_video, low_resolution_video, vectorized_video, khan_excercise...
-    """
-    available = models.BooleanField(default=False)
-    format_size = models.IntegerField(blank=True, null=True)
-    quality = models.CharField(max_length=50, blank=True, null=True)
-    contentmetadata = models.ForeignKey(ContentMetadata, related_name='formats', blank=True, null=True)
-    mimetype = models.ForeignKey(MimeType, blank=True, null=True)
+class FormatPreset(AbstractContent):
+    name = models.CharField(max_length=400, blank=True, null=True)
```

Comment:
Shouldn't allow name to be blank (also, even if we did, on `CharField` and `TextField`, you shouldn't set `null=True`)